### PR TITLE
added info about Device Registration Service

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5260

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 1]
```diff
@@ -366,6 +367,10 @@ Active Directory Federation Server used for Windows Hello for Business certifica
 Approximately 60 days prior to enrollment agent certificate’s expiration, the AD FS service attempts to renew the certificate until it is successful.  If the certificate fails to renew, and the certificate expires, the AD FS server will request a new enrollment agent certificate.  You can view the AD FS event logs to determine the status of the enrollment agent certificate.
 
 ### Service Connection Point (SCP) in Active Directory for ADFS Device Registration Service
+
+>[!NOTE]
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: MD compatibility space
```suggestion
> [!NOTE]
```

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 2]
```diff
@@ -366,6 +367,10 @@ Active Directory Federation Server used for Windows Hello for Business certifica
...
+> For detailed information about Device Registration Service see [Configuring Device Registration](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn614658(v=ws.11)?redirectedfrom=MSDN)
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: grammar details
```suggestion
> For detailed information about the Device Registration Service, see [Configuring Device Registration](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn614658(v=ws.11)?redirectedfrom=MSDN)
```

**Comment by e0i** [comment]
@MaratMussabekov Please resolve the merge conflicts to move forward with the review. Thanks.

**Comment by illfated** [comment]
(PR continued in #5541 )

**Pull request closed without merging**